src/services/nnid/routes/people.ts

```diff
@@ -610,9 +611,8 @@ router.put('/@me', async (request: express.Request, response: express.Response):
 	pnid.region = region;
 	pnid.timezone.name = timezoneName;
 	pnid.timezone.offset = Number(timezone.utc_offset);
-	pnid.timezone.marketing = marketingFlag;
-	pnid.timezone.off_device = offDeviceFlag;
-
+	pnid.flags.marketing = marketingFlag;
+	pnid.flags.off_device = offDeviceFlag;

```

Comment:
Please add a space between this and the data save

While you are here, can you also make this update some missing fields that are retreived above, like `language` and `countryCode`?